docs/source/notebooks/probabilistic_matrix_factorization.ipynb

```diff
@@ -0,0 +1,1489 @@
+{
```

Comment:
<p>If you run this again, can you bump to PyMC3 v3.8?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3709/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>